edx_proctoring/models.py

```diff
@@ -321,6 +324,58 @@ def get_active_student_attempts(self, user_id, course_id=None):
 
         return self.filter(filtered_query).order_by('-created')  # pylint: disable=no-member
 
+    def get_last_relevant_proctored_practice_attempt(self, user_id, proctoring_backend, course_id=None):
+        """
+        Filter user's proctored practice attempts within the last two years, and return one of the
+        following:
+            1. The last proctored practice attempt, if it exists
+            2. The most recent attempt, if there is no verified attempt
```

Comment:
Why bother return the most recent not verified attempt? Why have this function be generic to return last relevant attempt, instead of last verified attempt? This makes the function specificity mixed.